core/internal/runtime/bindings.go

```diff
@@ -146,8 +145,8 @@ func putBinding(ctx context.Context, kind string, actor Actor, id, payload strin
 	if len(keys) > 0 { // reuse existing key
 		key = keys[0]
 		successCode = http.StatusOK
-	} else { // new key with random partition
-		p := rpc.ChoosePartition()
+	} else { // new key using partition 0
+		p := 0
```

Comment:
Not thrilled about embedding a 0 here, but it was expedient. Abstract this into a function? Mark with a todo?  Don't worry about it?

Reply:
pushed this function back into the rpcLib API so we don't lose track of it.